packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -120,17 +120,35 @@ function registerModules(
     });
 }
 
+/**
+ * Reset the app state.
+ *
+ * @param container the UShell container
+ */
+async function resetAppState(container: typeof sap.ushell.Container): Promise<void> {
+    const appStateService = await container.getServiceAsync<AppState>('AppState');
+    const urlParams = new URLSearchParams(window.location.hash);
+
+    const appState = 'sap-iapp-state';
+
+    const appStateValue = urlParams.get(appState);
```

Comment:
Depending on position in the hash the key for the AppState is either `sap-iapp-state` or `/?sap-iapp-state`. So please change to
```suggestion
    const appStateValue = urlParams.get('sap-iapp-state') ?? urlParams.get('/?sap-iapp-state');
```

Reply:
Done and fixed with [1431102](https://github.com/SAP/open-ux-tools/pull/1825/commits/14311028e5452915a514f88b0e9447871923a244), thanks.